fed/src/sass/components.enhanced/ui/_organism-fancynav.scss

```diff
@@ -0,0 +1,36 @@
+.FancyNav {
+    $block: unquote("#{&}");
+
+    position: fixed;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    transform: translateX(100%);
+    transform: translate3d(100%, 0, 0);
+    transition: transform .3s ease, opacity .3s ease, visibility .3s ease, z-index .4s ease;
+    background-color: $color-white;
+    background-color: rgba($color-white, .95);
+    opacity: 0;
+    overflow: hidden;
+    visibility: hidden;
+    z-index: -100;
+
+    &.is-open {
+        transform: translateX(0);
+        transform: translate3d(0, 0, 0);
+        opacity: 1;
+        visibility: visible;
+        z-index: 300;
+        overflow-y: scroll;;
+    }
+
+    &-toggle {
+        position: absolute;
+        right: 0;
+        width: scale(delta);
+        height: scale(delta);
```

Comment:
Basically, in variables.scss I have a sass map callsed `$scale` with the various size values used throughout the site for type and spacing, and I wrote this function to grab it and either return a rem/em value, or fallback to a px value for the basic stylesheets.

```
@function scale($key, $unit: "rem") {
    $scale: map-get($scale, $key);

    @if $enhanced == true {
        @if $unit == "rem" {
            @return rem($scale);
        } @elseif $unit == "em" {
            @return em($scale);
        }
    } @else {
        @return unquote("#{$scale}px");
    }
}
```